resources/assets/tinker/lang/fr_FR.lang

```diff
@@ -538,402 +538,402 @@ tile.fluid.molten.enderium.name=Enderium fondu
-smeltery.brick.tooltip1=Smeltery structure block
-smeltery.brick.tooltip2=(Safe for decoration)
-smeltery.castingtable.tooltip=Basic Smeltery equipment
-smeltery.castingfaucet.tooltip=This Smeltery is leaking
-smeltery.castingbasin.tooltip=For the ambitious Smeltery
+smeltery.brick.tooltip1=Bloc de structure de fonderie
+smeltery.brick.tooltip2=(Bon pour la décoration)
+smeltery.castingtable.tooltip=Équipement de fonderie de base
+smeltery.castingfaucet.tooltip=Cette fonderie fuit
+smeltery.castingbasin.tooltip=Pour les fonderie ambitieuse
 #Dear translators, only translate these lines if the jokes are applicable in your language.
-grout.tooltip=I am Grout!
-frypankill.tooltip=This poor animal met its demise with a loud *bang*
-
-tconstruct.waila.empty=Empty
-tconstruct.waila.liquidtag=Liquid: 
-tconstruct.waila.amounttag=Amount: 
-tconstruct.waila.contains=Contains: 
-tconstruct.waila.subtanks=Subtanks:
-tconstruct.waila.invalidstructure=Invalid structure
-
-tconstruct.nei.toolmaterials=Tool Materials
-tconstruct.nei.projectilematerials=Bow & Arrow
-tconstruct.nei.alloying=Smeltery Alloying
-tconstruct.nei.castingbasin=Casting Basin
-tconstruct.nei.castingtable=Casting Table
-tconstruct.nei.dryingrack=Drying Rack
-tconstruct.nei.melting=Smeltery Melting
-
-gui.smeltery.fuel=Fuel
-gui.smeltery.emerald=Emeralds: 
-gui.smeltery.glass.block=Blocks: 
-gui.smeltery.glass.pannel=Panels: 
-gui.smeltery.metal.ingot=Ingots: 
-gui.smeltery.metal.nugget=Nuggets: 
-gui.smeltery.molten.check=Molten
-gui.landmine=Landmine
-gui.partcrafter1=Tool Part Crafting
-gui.partcrafter2=Tool Part Building
-gui.partcrafter3=Place a pattern and a material on the left to get started.
-gui.partcrafter4=Base Durability: 
-gui.partcrafter5=Handle Modifier: 
-gui.partcrafter6=Mining Speed: 
-gui.partcrafter7=Mining Level: 
-gui.partcrafter8= Heart
-gui.partcrafter9= Hearts
-gui.partcrafter10=Attack: 
-gui.partcrafter11=Full Durability: 
-gui.stenciltable1=Next Pattern
-gui.stenciltable2=Previous Pattern
-gui.toolforge1=Repair and Modification
-gui.toolforge2=The main way to repair or change your tools. Place a tool and a material on the left to get started.
-gui.toolstation1=Durability:
-gui.toolstation2=Durability: 
-gui.toolstation3=Attack: 
-gui.toolstation4=Bonus: 
-gui.toolstation5=Loss: 
-gui.toolstation6=Draw Speed: 
-gui.toolstation7=Arrow Speed: 
-gui.toolstation8=Weight: 
-gui.toolstation9=Accuracy: 
-gui.toolstation10=Base Damage:
-gui.toolstation11=Shortbow Attack:
-gui.toolstation12=Mining Speeds: 
-gui.toolstation13=Harvest Levels:
-gui.toolstation14=Mining Speed: 
-gui.toolstation15=Mining Level: 
-gui.toolstation16=Usage Speed: 
-gui.toolstation17=Modifiers
-gui.toolstation18=Modifiers remaining: 
-gui.toolstation19=Damage Reduction: 
-gui.toolstation20=Protection: 
-gui.toolstation21=Ammo: 
-gui.toolstation22=Break Chance: 
-gui.toolstation23=Max. Attack: 
-
-gui.mining1=Stone
-gui.mining2=Iron
+grout.tooltip=J'enduit !
```

Comment:
I believe this part should be changed back.  The grout part refers to Groot from Guardians from the Galaxy who constantly says "I am Groot!"  However, this doesn't work since grout translates to enduit in French.